packages/table/src/Table.ts

```diff
@@ -363,11 +363,15 @@ export class Table extends SizedMixin(SpectrumElement, {
         };
         if (!rowItem.value) {
             const { checkbox } = target as TableCheckboxCell;
-            if (checkbox.checked || checkbox.indeterminate) {
-                this.selectAllRows();
-            } else {
-                this.deselectAllRows();
+            if (checkbox) {
+                if (checkbox.checked || checkbox.indeterminate) {
+                    this.selectAllRows();
+                } else {
+                    this.deselectAllRows();
+                }
             }
+
+            return;
```

Comment:
Not 100%, but pretty sure you want something closer to:
```
if (!checkbox) return;

// ... rest of code here
```
But, I'd have to dig deeper and run the workflow to confirm.

Reply:
Working well